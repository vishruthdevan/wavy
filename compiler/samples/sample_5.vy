let load = fn(file) {
    puts("Loading file: " + file);
};

let changeFormat = fn(audio, format) {
    puts("Changing format of audio to: " + format);
};

let export = fn(audio, outputFile) {
    puts("Exporting audio to: " + outputFile);
    return outputFile;
};

let aud1 = "input1.wav";
let format = "aac";

load(aud1);
changeFormat(aud1, "aac");
let output1 = export(aud1, "output1.aac");

puts(output1)
output1