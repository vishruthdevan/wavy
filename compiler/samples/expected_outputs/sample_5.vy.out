0000 OpClosure 1 0
0004 OpSetGlobal 0
0007 OpClosure 3 0
0011 OpSetGlobal 1
0014 OpClosure 5 0
0018 OpSetGlobal 2
0021 OpConstant 6
0024 OpSetGlobal 3
0027 OpConstant 7
0030 OpSetGlobal 4
0033 OpGetGlobal 0
0036 OpGetGlobal 3
0039 OpCall 1
0041 OpPop
0042 OpGetGlobal 1
0045 OpGetGlobal 3
0048 OpConstant 8
0051 OpCall 2
0053 OpPop
0054 OpGetGlobal 2
0057 OpGetGlobal 3
0060 OpConstant 9
0063 OpCall 2
0065 OpSetGlobal 5
0068 OpGetBuiltin 1
0070 OpGetGlobal 5
0073 OpCall 1
0075 OpPop
0076 OpGetGlobal 5
0079 OpPop
